ledis/Controllers/ViewController.swift

```diff
@@ -12,14 +12,8 @@ class ViewController: UIViewController {
 
```

Comment:
if it's empty you can just delete it.
And when time comes and you realize that you need this controller for sth you can always create it again